src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -357,15 +361,31 @@ private void SaveSettingsCollectionStylesCss()
 			}
 		}
 
+		private void AddRtlCreditsCssRules(StringBuilder sb)
```

Comment:
Put a class in where? I need to put a class in based on whether this setting has rtl checked. But if I put it in the xmatter html, it'll get wiped out.